ubuntu/standard/5.0/Dockerfile

```diff
@@ -203,8 +203,10 @@ RUN set -ex \
 
 ENV NODE_12_VERSION="12.22.2"
```

Comment:
We should drop Node version 12 because Active Support ended 1 year and 6 months ago
(20 Oct 2020) and Security Support ended 2 weeks and 3 days ago
(30 Apr 2022).

Reply:
AWS Lambda still [supports Node.js 12](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html) and it's not on the [deprecation list](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtime-support-policy).  In any case: I think dropping Node 12 is out of scope for this PR.